pkg/core/transactor/listener.go

```diff
@@ -416,6 +418,16 @@ func (t *Transactor) handleUnconfirmedBalance(r rpcbus.Request) error {
 	return nil
 }
 
+func (t *Transactor) handleIsWalletLoaded(r rpcbus.Request) error {
+	if t.w == nil {
+		r.RespChan <- rpcbus.Response{*bytes.NewBufferString("not loaded"), nil}
```

Comment:
nitpick: It's more idiomatic if the producer (transactor) returns true/false as bool flag and consumers (rpc etc) translates this into a message ("not loaded"/"loaded")

Reply:
Gotcha, adding it.